src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Unix.cs

```diff
@@ -26,7 +26,8 @@ private static bool GetGlobalizationInvariantMode()
                     if (loaded == 0 && !OperatingSystem.IsBrowser())
                     {
                         string message = "Couldn't find a valid ICU package installed on the system. " +
-                                        "Set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support.";
+                                         "Please install libicu using your package manager and try again. " +
+                                         "Alternatively you can set the configuration flag System.Globalization.Invariant to true if you want to run with no globalization support.";
```

Comment:
Pre-existing your change, but this doesn't need to be in the resx?

Reply:
No we cannot have this in resx. Resources need globalization. If ICU is missing then all globalization would be broken. We usually use hardcoded message for the things we fail fast with. one example when there is something wrong in the builds which cause recursive lookup for resources. at that time we just fail fast with hardcoded message.

Reply:
Can we add a comment here to that effect, highlighting that it's an explicit choice to not put this in resources?